onnxruntime/core/framework/murmurhash3.cc

```diff
@@ -115,10 +115,10 @@ ORT_FORCEINLINE constexpr uint64_t fmix64(uint64_t k) {
 //-----------------------------------------------------------------------------
 
 namespace onnxruntime {
-void MurmurHash3::x86_32(const void* key, int len,
+void MurmurHash3::x86_32(const void* key, size_t len,
                          uint32_t seed, void* out) {
   const uint8_t* data = (const uint8_t*)key;
-  const int nblocks = len / 4;
+  const auto nblocks = static_cast<ptrdiff_t>(len / 4);
```

Comment:
nit: it is better to have `static_cast<ptrdiff_t>(len / 4U)` to indicate 4 is unsigned. Or use `static_cast<ptrdiff_t>(len >> 2)`